C/casacorewrapper/build_tarballs.jl

```diff
@@ -0,0 +1,46 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "casacorewrapper"
+version = v"0.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/kiranshila/casascorewrapper.git", "d7574e8639c8ac55a360652e4e799a4686bec93c")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/casascorewrapper/
+mkdir build && cd build
+cmake \
+    -DCMAKE_INSTALL_PREFIX=$prefix \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DCASACORE_ROOT_DIR=${prefix} \
+    ..
+make -j${nproc}
+make install
+"""
+
+# Use the same platforms from casacore
+platforms = supported_platforms(exclude=(platform)-> Sys.iswindows(platform) || Sys.isfreebsd(platform))
+platforms = expand_cxxstring_abis(platforms)
+# Expand libgfortran versions because we need to exclude some specific combinations
+platforms = expand_gfortran_versions(platforms)
+filter!(p -> libc(p) != "musl", platforms)
+filter!(p -> !(arch(p) == "powerpc64le" && libgfortran_version(p) == v"3"), platforms)
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("libcasacorewrapper", :libcasacorewrapper)
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = [
+    Dependency(PackageSpec(name="casacore_jll", uuid="72fd12c2-f19b-5d3c-931a-6bbe5223e3ce"))
+]
+
+# Build the tarballs, and possibly a `build.jl` as well.
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.6", preferred_gcc_version = v"6.1.0")
```

Comment:
Ok, this is a bit annoying. Since for powerpc we have only libgfortran 4 and 5, in order to match the ABI here we need to require gcc 7 (see https://github.com/JuliaPackaging/Yggdrasil/blob/3acc02f99ceaf85b0ada98745d9fc42dd9d6566f/RootFS.md#compiler-shards)
```suggestion
build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.6", preferred_gcc_version = v"7")
```

Reply:
Oof